docker/Dockerfile

```diff
@@ -281,17 +282,31 @@ RUN set -e; \
     git clone https://github.com/xkszltl/roaster.git /etc/roaster/scripts; \
     truncate -s0 ~/.bash_history;
 
+# [PEP 668](https://peps.python.org/pep-0668/) disallows installing pip packages directly to the base environment.
+# However, there are no viable alternatives, so we set `--break-system-package` as a workaround.
 RUN set -e; \
+    PIP_MAYBE_BREAK_SYSTEM_PACKAGES="$(set -e; \
```

Comment:
Do not name var with PIP_ because pip itself uses this namespace.
Recommend to just copy-paste a oneliner command in $() everywhere.

Reply:
That also looks bad to me...